src/ui/scss/component/_search.scss

```diff
@@ -53,7 +41,7 @@
 
 .search__options-wrapper {
   font-size: 1.25em;
-  padding-bottom: var(--spacing-vertical-large);
+  margin-bottom: var(--spacing-vertical-large);
```

Comment:
Please continue to do this. @NetOperatorWibby is on the same page that we should typically be using margin over padding without a strong reason to prefer padding.